src/gsy_e/models/area/scm_manager.py

```diff
@@ -301,13 +314,27 @@ def __init__(self, area: "CoefficientArea", time_slot: DateTime):
         SCMCommunityValidator.validate(community=area)
 
         self._home_data: Dict[str, HomeAfterMeterData] = {}
-        # Community is always the root area in the context of SCM.
-        self._community_uuid = area.uuid
+
+        self._community_uuid = self._get_community_uuid_from_area(area)
         self.community_data = CommunityData(self._community_uuid)
         self._time_slot = time_slot
         self._bills: Dict[str, AreaEnergyBills] = {}
         self._grid_fees_reduction = ConstSettings.SCMSettings.GRID_FEES_REDUCTION
 
+    def _get_community_uuid_from_area(self, area):
+        # Community is always the root area in the context of SCM.
+        # The following hack in order to support the UI which defines the Community one level
+        # lower than the Grid area.
```

Comment:
Nit: I now understand the rationale of this PR. However, a few weeks ago we had a conversation with Arslan and we agreed that the FE would have changed the 4-hierarchy logic as soon as possible. I would personally do this rather than implementing hacks in the backend, because they are likely to become technical debt as soon as we move on to implementing new features. Also, the ticket that the FE agree to create will now have 2 dependencies on the backend too ([gsy-framework](https://github.com/gridsingularity/gsy-framework/blob/c2804443c4a3f6f108311f60a4065e4c43feaa91/gsy_framework/community_datasheet/community_datasheet_parser.py#L114) and gsy-e here)

Reply:
I agree with you, however we need to deploy. If you want I can add a TODO / FIXME comment to remember. The UI has already a ticket open to deal with this IIRC , so it will not be neglected

Reply:
I couldn't find the ticket you mention. Can you share a link? Otherwise I'll ask the FE team to create it.

Reply:
You are right, I also cannot find the ticket so I might be wrong. If you can remind Arslan to add one then it would be great, thanks in advance!